src/components/FinancialInfo.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Typography, Button, Divider } from '@material-ui/core';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import TextField from './TextField';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      color: theme.palette.text.primary,
+    },
+    title: {
+      color: theme.palette.text.secondary,
```

Comment:
since this is being applied to a typography component, you can just remove this styling completely and instead write the component as `<Typography variant="h2" color="textSecondary"/>`